doc/README.md

```diff
@@ -14,11 +14,11 @@ git submodule update --init
 
 Now to build both SciPy itself and the docs, use:
 ```
-python3 runtests.py --doc html
+python3 dev.py doc
 ```
 
 Alternatively, if you prefer to build SciPy and the docs separately rather
-than use `runtests.py`:
+than use `dev.py`:
 ```
 python setup.py develop  # in the root of the repo
```

Comment:
setup.py develop (while we're at it)

Reply:
Maybe just remove this completely, there's no point having two sets of commands and `setup.py develop` is going to go away before the next release anyway.

Reply:
Yup I was hesitant on that, but I'm +1 for removing it